index.js

```diff
@@ -0,0 +1,55 @@
+const moles = document.querySelectorAll('.mole');
+const holes = document.querySelectorAll('.hole');
+const scoreTable = document.getElementById('score');
+const button = document.getElementById('button');
+const game = document.getElementById('game');
+let score = 0;
+let isTimeOver = false;
+let currMole;
+
+const chouseMole = (moles) => {
+  const index = Math.floor(Math.random() * moles.length);
+  const mole = moles[index];
+  if (mole === currMole) {
+    return chouseMole(moles);
+  }
+  currMole = mole;
+  return mole;
+};
+
+const showMole = () => {
+  const DISPLAY_TIME = 700;
```

Comment:
Why do you need to create this constant every time you show a mole? That is a constant. Will it be reasonable to define it only once?